gradle.properties

```diff
@@ -13,3 +13,9 @@ archives_base_name=bed-wars
 
 # Dependencies
 fabric_version=0.28.3+1.16
+
+# check this on https://maven.nucleoid.xyz/xyz/nucleoid/plasmid/
+plasmid_version=0.4.131
+
+# check this on https://maven.nucleoid.xyz/xyz/nucleoid/substrate/
```

Comment:
These comments could reference https://nucleoid.xyz/use/ rather than the maven :P

Reply:
Didn't know about this :D